noir-projects/aztec-nr/aztec/src/oracle/random.nr

```diff
@@ -6,21 +6,5 @@ pub unconstrained fn random() -> Field {
     rand_oracle()
 }
 
-/// Returns as many random bytes as specified through N.
-pub unconstrained fn get_random_bytes<let N: u32>() -> [u8; N] {
```

Comment:
As Mike pointed out, we should keep oracle folder for oracles. For this reason I moved this function to utils/random.nr without doing any changes to it.